javascriptv3/example_code/s3/README.md

```diff
@@ -1,82 +1,153 @@
-  in the *AWS Identity and Access Management User Guide*.
-- This code has not been tested in all AWS Regions. Some AWS services are
-  available only in specific AWS Regions. For more information, see the
-  [AWS Regional Services List](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
-  on the AWS website.
-- Running this code might result in charges to your AWS account.
+Code examples that show you how to accomplish a specific task by calling multiple
+functions within the same service.
 
-## Running the code
+* [Create a presigned URL](scenarios/presigned-url-upload.js) 
+* [Get started with buckets and objects](scenarios/basic.js) 
+* [Upload or download large files](scenarios/multipart-upload.js) 
+
+## Run the examples
 
 ### Prerequisites
-- An AWS account. To create an account, see [How do I create and activate a new AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) on the AWS Premium Support website.
-- AWS credentials. For details, see  [Setting credentials in Node.js](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-credentials-node.html) in the
-  *AWS SDK for Javascript (v3) Developer Guide*.
 
-1. Clone the [AWS SDK Code Samples repo](https://github.com/awsdocs/aws-doc-sdk-examples) to your local environment. See [the Github documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) for instructions.
 
-2. Install the dependencies listed in the package.json.
+For prerequisites, see the [README](../../README.md#Prerequisites) in the `javascriptv3` folder.
 
-```
-npm install node -g
-cd javascriptv3/example_code/s3
-npm install
-```
-3. In your text editor, update user variables specified in the ```Inputs``` section of the sample file.
 
-4. Run sample code:
+
+<!--custom.prerequisites.start-->
+<!--custom.prerequisites.end-->
+
+### Instructions
+
+**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see
+[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-examples-javascript-syntax.html).
+
+**Run a single action**
+
+```bash
+node ./actions/<fileName>
 ```
-cd src
-node [example name].js
+
+**Run a scenario**
+Most scenarios can be run with the following command:
+```bash
+node ./scenarios/<fileName>
 ```
 
-## Unit tests
-For more information see, the [README](../README.rst).
+<!--custom.instructions.start-->
+Some scenarios are web applications that must be bundled. These scenarios are in the `scenarios/web` folder.
+Follow these steps to run a web scenario. Some scenarios may require extra steps to get them working.
+
+1. Navigate to the web scenario you would like to run
+1. Run `npm i`
+1. Run `npm run dev`
+<!--custom.instructions.end-->
+
+
+#### Create a presigned URL
+
+This example shows you how to create a presigned URL for S3 and upload an object.
```

Comment:
```suggestion
This example shows you how to create a presigned URL for Amazon S3 and upload an object.
```

Reply:
@lkdavies this one is auto generated by WRITEME. It's pulling from the same metadata that SOS pulls from. Leaving it for now because I think it's accurate.